README.md

```diff
@@ -56,6 +56,7 @@
 `biomass_soil_[season]_[crop].csv` (`biomass_soil_winter_SH.csv`)
 - `Location`: experimental plot location at TREC [*linking field*]
 - `CropTrt`: experimental treatment denoting cover crops in monoculture or mixture [*linking field*]
+- 'CropName': abbreviated name of individual cover crops e.g. SH for sunn hemp
```

Comment:
These quotes `'` should be backticks `` ` `` .

Reply:
oops! I just fixed that. Took me a while to figure out where the back ticks were haha